### PR TITLE
Changing behaviour on unsupported OS

### PR DESCRIPTION
**Pull request by saadrahim**
I think the default behaviour needs changing. Please let me know what you think.

### CODE REVIEW

File: share/rocm/cmake/ROCMCreatePackage.cmake [hunk 1]
```diff
@@ -40,7 +40,8 @@ macro(rocm_create_package)
     elseif(_os_id_sles)
```
Comment (saadrahim) [inline share/rocm/cmake/ROCMCreatePackage.cmake]: Maybe I should remove this as SLES is part of the default case.

File: share/rocm/cmake/ROCMCreatePackage.cmake [hunk 2]
```diff
@@ -37,10 +37,9 @@ macro(rocm_create_package)
     #Debs use underscrore between OS and architecture
     elseif(_os_id_ubuntu)
         STRING(CONCAT _SYSTEM_NAME ${_os_id} "-" ${_version_id} "_amd64")
-    elseif(_os_id_sles)
-        STRING(CONCAT _SYSTEM_NAME ${_os_id} "-" ${_version_id} ".amd64")
     else()
-        set(_SYSTEM_NAME ${CPACK_SYSTEM_NAME})
+    #For SLES and unsupported OSes
+        STRING(CONCAT _SYSTEM_NAME ${_os_id} "-" ${_version_id} ".amd64")
     endif()
```
Comment (saadrahim) [inline share/rocm/cmake/ROCMCreatePackage.cmake]: All modern Linux distributions have /etc/os-release. What should our behaviour be if it does not exist?

Comment (pfultz2) [reply share/rocm/cmake/ROCMCreatePackage.cmake]: We should just use `CMAKE_SYSTEM_NAME` when ` /etc/os-release` is not found.

Comment (saadrahim) [reply share/rocm/cmake/ROCMCreatePackage.cmake]: Please do not merge yet. I fixed it to go with default behavior. I am going to test this out and see.

**Comment by pfultz2** [comment]
@saadrahim Is this ready to merge?

**Comment by saadrahim** [comment]
Please merge, @amdkila tested it.

**Comment by saadrahim** [comment]
HI @pfultz2,
Can you please merge this?

Saad

**Pull request merged**